lib/node_modules/@stdlib/assert/has-same-constructor/docs/repl.txt

```diff
@@ -0,0 +1,35 @@
+{{alias}}( value )
+    Tests if two values have same constructor.
+
+    If provided with two values with different constructors, the function returns `false`.
+
+    Parameters
+    ----------
+    value: any
+        Value to test.
+
+    Returns
+    -------
+    bool: boolean
+        Boolean indicating whether two values have same constructor.
+
+    Examples
+    --------
+    > var obj1 = new Date();
```

Comment:
This is not how we do examples. See https://github.com/stdlib-js/stdlib/blob/develop/docs/repl_text.md.